### PR TITLE
bbrecon: new tool

### PR DESCRIPTION
**Pull request by gkroon**
I've created a [new tool](https://gitlab.com/0bs1d1an/bbrecon), inspired by [@nahamsec's lazyrecon](https://github.com/nahamsec/lazyrecon), but the original was too difficult for me to package so I basically refactored it myself (apart from the report output function). Can we add this  to the tree? Please let me know if anything needs changing. I'm also accepting merge requests for bbrecon itself.

### CODE REVIEW

File: profiles/pentoo/base/package.accept_keywords/app-text [hunk 1]
```diff
@@ -7,3 +7,6 @@ app-text/cewl
 
 ~kde-apps/okular-backend-mupdf-0.0.5
 =kde-apps/okular-15.12*
+
+#required by net-analyzer/bbrecon
+app-text/unfurl-0.2.0
```
Comment (linxon) [inline profiles/pentoo/base/package.accept_keywords/app-text]:
```
RepoMan scours the neighborhood...
--- Invalid atom in /home/linxon/GitClones/pentoo-overlay/profiles/pentoo/base/package.accept_keywords/app-text: app-text/unfurl-0.2.0
--- Invalid atom in /home/linxon/GitClones/pentoo-overlay/profiles/pentoo/base/package.accept_keywords/app-text: app-text/unfurl-0.2.0
--- Invalid atom in /home/linxon/GitClones/pentoo-overlay/profiles/pentoo/base/package.accept_keywords/app-text: app-text/unfurl-0.2.0
--- Invalid atom in /home/linxon/GitClones/pentoo-overlay/profiles/pentoo/base/package.accept_keywords/app-text: app-text/unfurl-0.2.0
--- Invalid atom in /home/linxon/GitClones/pentoo-overlay/profiles/pentoo/base/package.accept_keywords/app-text: app-text/unfurl-0.2.0
```

File: profiles/pentoo/base/package.accept_keywords/net-misc [hunk 2]
```diff
@@ -119,3 +119,7 @@ net-misc/voipong
 =net-misc/seafile-client-7*
 =net-misc/seafile-7*
 =net-libs/libsearpc-3*
+
+#required by net-analyzer/bbrecon
+~net-misc/httprobe-0.1.1
```
Comment (linxon) [inline profiles/pentoo/base/package.accept_keywords/net-misc]: 0.1.2 was added

File: profiles/pentoo/base/package.accept_keywords/app-text (hunk 1 above)
Comment (gkroon) [reply profiles/pentoo/base/package.accept_keywords/app-text]: Fixed

File: profiles/pentoo/base/package.accept_keywords/net-misc (hunk 2 above)
Comment (gkroon) [reply profiles/pentoo/base/package.accept_keywords/net-misc]: Fixed

**Comment by linxon** [comment]
thank you :)

**Pull request merged**